WordPress/Classes/Services/MediaRepository.swift

```diff
@@ -90,14 +90,14 @@ private extension MediaRepository {
 
     @objc(remoteForBlog:error:)
     public func remote(for blog: Blog) throws -> MediaServiceRemote {
-        if blog.supports(.wpComRESTAPI), let dotComID = blog.dotComID, let api = blog.wordPressComRestApi {
-            return MediaServiceRemoteREST(wordPressComRestApi: api, siteID: dotComID)
-        }
-
         if let site = try? WordPressSite(blog: blog) {
             return MediaServiceRemoteCoreREST(client: .init(site: site))
         }
```

Comment:
Just a reminder to myself that we'll keep the original implementation for media, where we prefer the WP.com API over the REST API.